.github/workflows/ci-failure-autofix.yml

```
@@ -0,0 +1,253 @@
+          # Apply with --3way so minor context differences between master and
+          # the branch don't cause a hard failure.
+          if ! git apply --3way --index /tmp/autofix.patch; then
+            echo "::warning::Verified patch does not apply cleanly to branch tree — manual fix needed"
+            exit 0
+          fi
+
+          git add -A
+          # Commit message uses env var (safe — never ${{ }} in run:)
+          git commit -m "fix(ci-autofix): auto-fix CI failure on ${AUTOFIX_BRANCH}
+
+Patch generated by Claude Sonnet 4.6. All tests pass after this change.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)